src/Card/CardImageCap.jsx

```diff
@@ -130,6 +135,7 @@ CardImageCap.defaultProps = {
   logoSkeletonHeight: LOGO_SKELETON_HEIGHT_VALUE,
   skeletonWidth: undefined,
   logoSkeletonWidth: undefined,
+  imageLoadingType: undefined,
```

Comment:
Should the default value be `eager` here? This is what the browser will default to anyway, but still might be helpful to explicitly specify it here.

Reply:
Tried to add it but tests start failling due it will render with <CardImageCap> ..... loading={'eager'} </> I made the fix for the test snapshots  but that means users will also have the same test issue with the snapshots. You can check it [here](https://github.com/openedx/paragon/pull/2328/commits/e4687a49dd7829804e9cf19a3d26f69a9fdfdaa4). Let me know your thoughts.

Reply:
@ochavarria2u I agree about defaulting to `eager` in the `CardImageCap.defaultProps`. If the internals of a Paragon component changes, that shouldn't impact consumers' tests. If it does, I feel they should be testing in a different way that doesn't rely on Paragon's internals. All that to say, adding `loading="eager"` by default on `CardImageCap` here is an implementation detail of the component so should be fine to change imo.

Reply:
Got it, I just added a commit with the changes.